src/cloud/project/storage/cloud-project-storage.service.js

```diff
@@ -4,6 +4,9 @@ angular.module("ovh-api-services").service("OvhApiCloudProjectStorage", function
     return {
         v6: function () {
             return $injector.get("OvhApiCloudProjectStorageV6");
+        },
+        aapi: function () {
```

Comment:
In order to make consistency across all services which are injecting 2API calls! 

```suggestion
        Aapi: function () {
```